File: Courses/Bedu/DesarrolloMobileJava/Sesion6/Sesion6/app/src/main/java/xyz/eliothmonroy/sesion6/MainActivity.java
package xyz.eliothmonroy.sesion6;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void compartir(View view) {
        Intent intent=new Intent();
        intent.setAction(Intent.ACTION_SEND);
        intent.putExtra(Intent.EXTRA_TEXT,"Texto a compartir");
        intent.setType("text/plain");

        Intent shareIntent=Intent.createChooser(intent,null);
        startActivity(shareIntent);
    }
}

File: Coroutines/RestConsumeTest/app/src/main/java/xyz/eliothmonroy/restconsumetest/MainActivity.kt
package xyz.eliothmonroy.restconsumetest

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import xyz.eliothmonroy.restconsumetest.async.AsyncActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startActivity(Intent(this,AsyncActivity::class.java))
    }
}
